clarifai/runners/models/model_upload.py

```diff
@@ -59,11 +60,7 @@ def _load_config(config_file: str):
       config = yaml.safe_load(file)
     return config
 
-  def _validate_config_checkpoints(self):
-    if not self.config.get("checkpoints"):
-      logger.info("No checkpoints specified in the config file")
-      return None
-
+  def _validate_config_checkpoints(self) -> typing.Tuple[str, str]:
```

Comment:
I also created PR to takes care of this
https://github.com/Clarifai/clarifai-python/pull/417/files#diff-7ee953fca56b92c0015d093416f9a390d19331e5cea0e845f85dfddb6cef4117R65

Reply:
ok go ahead and merge that then.